server/coordinator.go

```diff
@@ -143,12 +143,14 @@ func (c *coordinator) patrolRegions() {
 
 			key = region.GetEndKey()
 
-			if c.checkRegion(region) {
-				break
-			}
+			c.checkRegion(region)
```

Comment:
can it cause other operations (not only for removing nodes) too fast?

Reply:
I don't think it is a good fix. As we have schedule limit, when there are many regions need to update, only operators for the first serval regions can be created. The rest will not be handled util scan them the next time.

There are 2 approaches could be better:
1) decrease sleep time to 10ms and make it configurable.
2) don't sleep if an operator is successfully created.